### ISSUE TITLE
In findExternal, context should be a string

### ISSUE THREAD

**Issue by Satyam** [benmosher/eslint-plugin-import#411]
(a suggestion I'm not sure about:)

In https://github.com/benmosher/eslint-plugin-import/blob/master/resolvers/webpack/index.js#L174 it might be safer if the second `null` argument corresponding to the 'context' argument is an empty string instead.  Webpack will always provide a string for this argument and no document says it will do otherwise.   Under that circumstance, it is normal to use string methods such as `indexOf` which will make it fail when a `null` is received.  An empty string, I believe,  serves the same purpose and doesn't break anything.

**Comment by benmosher** [comment]
What should the string value be, ideally? I'm not familiar with external functions, this feature came in via a PR. 

I suspect it's supposed to be a file path; could probably provide what is actually expected per Webpack spec.

**Comment by Satyam** [comment]
It is the absolute pathname to the folder containing the file with the `require`  call. For relative paths, it would allow you to use `path.resolve` to get the absolute path to the required file, or a path that can be further processed by `require.resolve` to add the usual defaults (`index.js` and such).

If you add the following to any `webpack.config.js` file you might have, you will see what it takes without disturbing the build process:

```js
externals: [
      function (context, request, callback) {
        console.log(context, request);
        if (request[0] === '.') {
          console.log('=>>', require.resolve(path.resolve(context, request)));
        }
        callback();
      }
],
```

I guess that from https://github.com/benmosher/eslint-plugin-import/blob/master/resolvers/webpack/index.js#L15-L26, using `path.dirname(file)` instead of `null` would produce the same result.  An extra argument to `findExternal` with this information would be needed.
